components/kyma-environment-broker/internal/provider/aws_provider.go

```diff
@@ -79,20 +88,114 @@ func ZoneForAWSRegion(region string) string {
 	if !found {
 		zones = "a"
 	}
+	rand.Seed(time.Now().UnixNano())
 	zone := string(zones[rand.Intn(len(zones))])
 	return fmt.Sprintf("%s%s", region, zone)
 }
 
+func MultipleZonesForAWSRegion(region string, zonesCount int) []string {
+	zones, found := awsZones[region]
+	if !found {
+		zones = "a"
+		zonesCount = 1
+	}
+	rand.Seed(time.Now().UnixNano())
+
+	availableZones := strings.Split(zones, "")
+	rand.Shuffle(len(availableZones), func(i, j int) { availableZones[i], availableZones[j] = availableZones[j], availableZones[i] })
+	availableZones = availableZones[:zonesCount]
```

Comment:
nit: do we want to ensure that `zonesCount` is less than `len(availableZones)`? If I understand this correctly, `zonesCount` may at some point in future be user input and this may panic if `zonesCount` end up being larger than the capacity of `availableZones`.